opcplc/nodesfile.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$type": "OpcPlc.ConfigFolder, opcplc",
+  "NamespaceId": 7,
+  "Name": "MyTelemetry",
+  "NodeList": {
+    "$type": "System.Collections.Generic.List`1[[OpcPlc.ConfigNode, opcplc]], System.Private.CoreLib",
+    "$values": [
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1023",
+        "Name": "ActualSpeed",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Rotational speed"
+      },
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1024",
+        "Name": "aRMS",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Mean of the vibration acceleration"
+      },
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1025",
+        "Name": "DKW",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Diagnostic characteristic value"
+      },
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1026",
+        "Name": "Peak value",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Peak value of vibration acceleration"
+      },
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1027",
+        "Name": "Vibration Velocity",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Mean of the vibration velocity"
+      },
+      {
+        "$type": "OpcPlc.ConfigNode, opcplc",
+        "NodeId": "1029",
+        "Name": "Active power",
+        "DataType": "Float",
+        "ValueRank": -1,
+        "AccessLevel": "CurrentReadOrWrite",
+        "Description": "Active power"
+      }
+    ]
+  }
+}
```

Comment:
here is an example with all modifications:
{
  "Name": "MyTelemetry",
  "NodeList": [
      {
        "NodeId": "1023",
        "Name": "ActualSpeed",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Rotational speed"
      },
      {
        "NodeId": "1024",
        "Name": "aRMS",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Mean of the vibration acceleration"
      },
      {
        "NodeId": "1025",
        "Name": "DKW",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Diagnostic characteristic value"
      },
      {
        "NodeId": "1026",
        "Name": "Peak value",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Peak value of vibration acceleration"
      },
      {
        "NodeId": "1027",
        "Name": "Vibration Velocity",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Mean of the vibration velocity"
      },
      {
        "NodeId": "1029",
        "Name": "Active power",
        "DataType": "Float",
        "ValueRank": -1,
        "AccessLevel": "CurrentReadOrWrite",
        "Description": "Active power"
      }
    ]
}